utils/update_changelog.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+# usage: ./update_changelog.sh <release number>
+release=$1
+cl=CHANGELOG.md
+today=$(date +%F)
+header=$(head -n7 ${cl})
+older=$(tail -n+7 ${cl})
+
+# get all commits on head since the latest tag
+changes=$(git log --no-merges --pretty=format:"- %s" $(git describe --tags --abbrev=0 HEAD^)..HEAD)
+
+echo "${header}" > ${cl}
```

Comment:
This seems to overwrite the changelog. So... we're not able to keep the older entries then?

Reply:
sorry the script is ugly (I bash program with a hammer), it does keep the older entries (see the echo of ${older} below)

Reply:
let's shipit then! :smile: